.github/workflows/inspektor-gadget.yml

```diff
@@ -1337,11 +1325,14 @@ jobs:
           sudo mv ig /usr/bin/ig
       - name: Build gadgets
         run: |
-          make \
-          GADGET_REPOSITORY=${{ steps.set-repo-determine-image-tag.outputs.gadget-repository }} \
-          GADGET_TAG=${{ steps.set-repo-determine-image-tag.outputs.gadget-tag }} \
-          BUILDER_IMAGE=${{ needs.build-helper-images.outputs.ebpf_builder_image }} \
-          push-gadgets -o install/ig
+          export GADGET_REPOSITORY=${{ steps.set-repo-determine-image-tag.outputs.gadget-repository }}
+          export GADGET_TAG=${{ steps.set-repo-determine-image-tag.outputs.gadget-tag }}
+          export BUILDER_IMAGE=${{ needs.build-helper-images.outputs.ebpf_builder_image }}
+
+          make GADGET_BUILD_PARAMS="--update-metadata" build-gadgets -o install/ig
+          # Check that metadata files are updated
+          git diff --exit-code HEAD --
+          make push-gadgets -o install/ig
```

Comment:
`make -C gadgets/ push` depends on [build](https://github.com/inspektor-gadget/inspektor-gadget/blob/507c93faabeb01a2ef83fede75bf595cc701c4d3/gadgets/Makefile#L43). So, we are building again the gadgets here, see the raw logs of [this job](https://github.com/inspektor-gadget/inspektor-gadget/actions/runs/8603565645/job/23575899853).

From a quick test `make push-gadgets -o install/ig,build` doesn't work. So, we probably need to do `make -C gadgets/ push -o build`.

Reply:
updated, thanks!